packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -470,7 +484,6 @@ describe('ReactDOMFloat', () => {
           <head />
           <body>
             foo
-            <link rel="preload" as="style" href="foo" />
```

Comment:
According to the test, its actions should be run on the server. But as far as I can tell, there's only a client dispatcher (`ReactDOMClientDispatcher`).

@gnoff Do you have a e2e test for this scenario so that we can verify where the test environment might not be setup properly?

Jest has become a lot stricter with mixing environments and these sort of tests break due to strict environment separation. IMO this is ok behavior from Jest since it makes it more obvious where particular code is running.